apps/prairielearn/src/migrations/20250210225102_tags__not_null_constraints__add.sql

```diff
@@ -0,0 +1,22 @@
+ALTER TABLE tags
+ALTER COLUMN name
+SET NOT NULL;
+
+ALTER TABLE tags
+ALTER COLUMN color
+SET NOT NULL;
+
+ALTER TABLE tags
+ALTER COLUMN description
+SET DEFAULT '';
+
+UPDATE tags
+SET
+  description = ''
+WHERE
+  description IS NULL;
+
+-- TODO: before deploying this, ensure all writes use a non-null description.
```

Comment:
We have `NULL` values in production. We need to update the existing syncing code to fallback to an empty string at sync time and deploy that to ensure that we don't get any constraint violations as this is being deployed.

Reply:
This was done in #11339, which has now been deployed.